pkg/virt-controller/watch/migration/migration_test.go

```diff
@@ -99,6 +100,19 @@ var _ = Describe("Migration watcher", func() {
 		}
 	}
 
+	getTargetPod := func(namespace string, uid types.UID, migrationUid types.UID) (*k8sv1.Pod, error) {
```

Comment:
Make it a regular func instead of lambda?

Reply:
done